jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1278,6 +1284,12 @@ public void setProperty(String key, String value, String source)
                 throw ue;
             }
         }
+
+        // to override default http://central.maven.org/maven2/
+        if (key.equals("maven-base-url"))
```

Comment:
Our other properties are dot separated rather that - separated.
I also don't think we need url in the name.  Just `maven.base` works for me!

Reply:
Some thoughts:

This isn't a "base", but a "uri" that all content is looked relative to.

Some example values:

* `https://central.maven.org/maven2/` - standard location
* `https://repo.corp.com/` - corporate repository as http
* `file://Z:/shared/maven/repo` - shared drive repository on windows as file reference
* `file:////repo.corp.com/maven2/` - windows UNC reference
* `file://opt/SAN/shared/maven/repo` - mounted network SAN drive on unix as file reference

So how about it being just `maven.repo.uri` ?

Reply:
@joakime yup makes sense. Done

Reply:
Done?  Have you pushed? I still see `maven-base-uri` rather than `maven.repo.uri`

Reply:
Note that I like `maven.repo.uri`, so once you've changed to that I'm good for you to squash and merge